tests/PerformanceHarness/example_trx_generator_run.md

```diff
@@ -0,0 +1,97 @@
+
+## Command
+Sleep is needed otherwise subjective transaction may reject transactions as too soon.
+```
+CHAIN_ID=$(cleos --url $HTTP_URL get info | grep chain_id | cut -d:  -f2 | sed 's/[ ",]//g')
+LIB_ID=$(cleos --url $HTTP_URL get info | grep last_irreversible_block_id | cut -d:  -f2 | sed 's/[ ",]//g')
+sleep 3
+${BUILD_TEST_DIR}/trx_generator/trx_generator --generator-id $GENERATORID \
+     --chain-id $CHAIN_ID \
+     --contract-owner-account eosio \
+     --accounts $COMMA_SEP_ACCOUNTS \
+     --priv-keys $COMMA_SEP_KEYS \
+     --last-irreversible-block-id $LIB_ID \
+     --log-dir $TRX_LOG_DIR \
+     --peer-endpoint-type p2p \
+     --peer-endpoint $P2P_HOST \
+     --port $PEER2PEERPORT
+```
+
+## Output
+Why is this 0us
```

Comment:
It is `0` because this is logging `_total_us` which is never updated. Just initialized to 0. Seems you found a bug. I don't guess anyone ever noticed.

Reply:
forgot I had that question, converted PR to draft. I'll continue to add documentation updates, and better document the why 0us. 

This example is from Jungle Spring Beta2 test net. I believe an intermediate node is filtering the submitted transactions to the P2P endpoint, and they are not making onto the blockchain. Need some additional tests to verify.

Reply:
Time is zero, because `_total_us` is never updated.